instagram/src/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Comment from './Comment';
+
+const CommentSection = (props) => {
+        return (
+            <div>
+                <p>{props.obj.username}</p>
+                <img src={props.obj.thumbnailUrl} />
+                <img src={props.obj.imageUrl} />
+                <p>{props.obj.likes}</p>
+                <p>{props.obj.timestamp}</p>
+
+                {props.obj.comments.map(item => {
+                    return (
+                        <Comment key={item.text} username={item.username} text={item.text} />
+                    )
+                })}
+
+                <input></input>
+
+            </div>
+        )
+}
+
```

Comment:
Try to add Prop Types to all places where props are passing down data

Reply:
I added Prop Types to just the PostContainer.js because it checks all through the data, including the comment array. However, I can add additional Prop Type checking in the children functional components if you recommend it?

Reply:
Your prop types look good! Since you type checked all the data including the comments, you are fine to continue. Sorry, I reviewed this before I saw you add the propTypes. Good job!